nbs/core.ipynb

```diff
@@ -35,7 +35,7 @@
   {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #209.            temporal_cols: List[str] = [],
</pre><p>Sorry I didn't specify where to put this. We can infer what the temporal cols should be from the models, we already have this method in core <a href="https://github.com/Nixtla/neuralforecast/blob/0b9fad9d4bea56f4f278687fb03cb424ce25eb6f/neuralforecast/core.py#L542" target="_blank">https://github.com/Nixtla/neuralforecast/blob/0b9fad9d4bea56f4f278687fb03cb424ce25eb6f/neuralforecast/core.py#L542-L545</a> and we use a similar logic to get the historic exog. Then the temporal_cols will be the union of those</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/1049/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
ah I see - I've now added this as the method _get_needed_temporal_cols in the _prepare_fit_for_local_files method